controllers/news.js

```diff
@@ -16,11 +18,33 @@ exports.getArticles = (req, res) => {
 	});
 };
 
-exports.getArticlesById = (req, res) => {
+exports.getArticlesById = (req, res, next) => {
 	const { article_id } = req.params;
-	selectArticlesById(article_id).then((returnedArticle) => {
-		returnedArticle === undefined
-			? res.status(404).send({ msg: 'Route not found' })
-			: res.status(200).send(returnedArticle);
-	});
+	selectArticlesById(article_id)
+		.then((returnedArticle) => {
+			res.status(200).send(returnedArticle);
+		})
+		.catch((err) => next(err));
+};
+
+exports.getArticleComments = (req, res, next) => {
+	const { article_id } = req.params;
+	selectArticleComments(article_id)
+		.then((returnedComments) => {
+			res.status(200).send(returnedComments);
+		})
```

Comment:
I would send back the comments as an object on a key of returned comments -> {returnedComments} 
This will mean in your test instead of checking body[0] you can check body.returnedComments. 
This will be much easier to deal with once you plug this into your front end project in a few weeks time! A nicely named key will be easier to find when you want to render this information 😄